packages/clerk-js/src/core/resources/SignIn.ts

```diff
@@ -168,13 +175,21 @@ export class SignIn extends BaseResource implements SignInResource {
   };
 
   public authenticateWithRedirect = async (params: AuthenticateWithRedirectParams): Promise<void> => {
-    const { strategy, redirectUrl, redirectUrlComplete, identifier } = params || {};
-    const { firstFactorVerification } = await this.create({
-      strategy,
-      identifier,
-      redirectUrl: SignIn.clerk.buildUrlWithAuth(redirectUrl),
-      actionCompleteRedirectUrl: redirectUrlComplete,
-    });
+    const { strategy, redirectUrl, redirectUrlComplete, identifier, continueSignIn } = params || {};
+
+    const { firstFactorVerification } = continueSignIn
```

Comment:
❓ How about removing this new extra param and using `this.id` to decide if the sign-in is existing or not?

Reply:
I'm worried this might break cases that create a new SI even thought one already exists.

Reply:
How about combining `this.id` with `strategy="saml"` for now? We should refrain from adding more parameters to the API until it's absolutely necessary.

Reply:
Sure

Reply:
Removed in favour of the check for `saml` strategy.